lib/dataLayer.tsx

```diff
@@ -229,6 +231,7 @@ async function _getFormByStatus(
                 formID: record.formID,
                 ...record.formConfig.form,
                 publishingStatus: record.formConfig.publishingStatus,
+                displayAlphaBanner: record.formConfig.displayAlphaBanner ?? true,
```

Comment:
there probably isn't a need to explicitly set it IMO. You could treat undefined as true. Not a big deal though